### PR TITLE
Fix FTBFS on mips64el architecture due to missing definition of the _…

### PR DESCRIPTION
**Pull request by sunweaver**
…XSERVER64 macro.

### CODE REVIEW

**Comment by sunweaver** [comment]
@Ionic: please take a look at this PR (#454). Tested against Debian experimental. See https://buildd.debian.org/status/package.php?p=nx-libs&suite=experimental#problem-1

File: nx-X11/config/cf/linux.cf [hunk 1]
```diff
@@ -733,6 +733,21 @@ XCOMM binutils:	(LinuxBinUtilsMajorVersion)
 # define ServerExtraDefines	-DGCCUSESGAS XFree86ServerDefines
 #endif
 
+#ifdef MipselArchitecture
+# ifndef OptimizedCDebugFlags
+#  define OptimizedCDebugFlags	DefaultGcc2MipsOpt
+# endif
+# define LinuxMachineDefines	-D__MIPSEL__
+# define ServerOSDefines	XFree86ServerOSDefines -DDDXTIME
+# ifdef Mips64elArchitecture
```
Comment (Ionic) [inline nx-X11/config/cf/linux.cf:741]: Did you take the `AsOutputArchSize` stuff from the `SparcArchitecture` block? It looks like we haven't ever used `AsOutputArchSize` in the first place, so may as well not include it in the Mipsel block...

Comment (Ionic) [reply nx-X11/config/cf/linux.cf:741]: You could have probably merged this into the `Mips` block and checked for `Mips(64)elArchitecture` when needed within it. Would have made some stuff more compact, but maybe also more difficult to read. Your call.

Comment (sunweaver) [reply nx-X11/config/cf/linux.cf:741]: > Did you take the AsOutputArchSize stuff from the SparcArchitecture block? It looks like we haven't ever used AsOutputArchSize in the first place, so may as well not include it in the Mipsel block...

You are right dropping AsOutputArchSize here, as it seems unused elsewhere. Also removing it from the Sparc part, as that was indeed the place I imitated.

Comment (sunweaver) [reply nx-X11/config/cf/linux.cf:741]: > You could have probably merged this into the Mips block and checked for Mips(64)elArchitecture when needed within it. Would have made some stuff more compact, but maybe also more difficult to read. Your call.

Leaving it as is.

**Comment by sunweaver** [comment]
@Ionic: #454 is ready for merging.

**Pull request merged**